frontend/src/components/LoginButtonModal.tsx

```diff
@@ -28,7 +28,8 @@ export const LoginButtonModal = ({
           <ModalToggleButton
             modalRef={modalRef}
             opener
-            className="usa-nav__link text-ysaprimary font-sans-2xs display-flex text-normal border-0"
+            className="usa-nav__link font-sans-2xs display-flex text-normal border-0"
```

Comment:
discovered this `text-ysaprimary` class that I don't think is doing anything, so removed